File: LottoPlayer/app/src/main/java/com/im/daeseong/lottoplayer/MainTab2Fragment.java
package com.im.daeseong.lottoplayer;

import android.os.Bundle;
import androidx.annotation.Nullable;//import android.support.annotation.Nullable;
import androidx.fragment.app.Fragment;//import android.support.v4.app.Fragment;
import androidx.recyclerview.widget.LinearLayoutManager;//import android.support.v7.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;//import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.content.Context;

public class MainTab2Fragment extends Fragment {

    private static final String TAG = MainTab2Fragment.class.getSimpleName();

    private Context mContext;
    private View MainView;

    private RecyclerView recyclerView;
    private LottoTopRecyclerAdapter lottoTopRecyclerAdapter;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {

        mContext = container.getContext();
        return inflater.inflate(R.layout.fragment_main_tab2, container, false);
    }

    @Override
    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        MainView = view;

        recyclerView = (RecyclerView)view.findViewById(R.id.recycler_view);
        lottoTopRecyclerAdapter = new LottoTopRecyclerAdapter(mContext, ((MainActivity)getActivity()).getLottoTop());
        recyclerView.setLayoutManager(new LinearLayoutManager(mContext));
        recyclerView.setAdapter(lottoTopRecyclerAdapter);
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();

    }
}

File: LottoPlayer(kotlin)/app/src/main/java/com/daeseong/lottoplayer/MainTab2Fragment.kt
package com.daeseong.lottoplayer

import android.content.Context
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.daeseong.lottoplayer.Database.LottoTop

class MainTab2Fragment : Fragment() {

    companion object {
        private val tag = MainTab2Fragment::class.java.simpleName
    }

    private lateinit var mContext: Context
    private lateinit var MainView: View
    private lateinit var recyclerView: RecyclerView
    private lateinit var lottoTopRecyclerAdapter: LottoTopRecyclerAdapter

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        mContext = container!!.context
        return inflater.inflate(R.layout.fragment_main_tab2, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        MainView = view
        recyclerView = view.findViewById(R.id.recycler_view)
        lottoTopRecyclerAdapter = LottoTopRecyclerAdapter(mContext, (activity as MainActivity?)!!.getLottoTop() as List<LottoTop>)
        recyclerView.layoutManager = LinearLayoutManager(mContext)
        recyclerView.adapter = lottoTopRecyclerAdapter
    }

    override fun onDestroyView() {
        super.onDestroyView()
    }
}
